packages/editor/src/components/Panel/Colors.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
```

Comment:
I think for starting this off it'd be better to keep all the components flat in the `src/` directory rather than nested in other directories – it looks like all the `Panel` stuff is the UI for the devtools, right?